ghost/core/core/server/services/mentions/BookshelfMentionRepository.js

```diff
@@ -54,7 +54,8 @@ module.exports = class BookshelfMentionRepository {
             sourceAuthor: model.get('source_author'),
             sourceExcerpt: model.get('source_excerpt'),
             sourceFavicon: model.get('source_favicon'),
-            sourceFeaturedImage: model.get('source_featured_image')
+            sourceFeaturedImage: model.get('source_featured_image'),
+            verified: model.get('verified')
```

Comment:
This doesn't currently work - we need to work out the best way to pass the `verified` property in